src/displayapp/screens/WatchFaceAnalog.cpp

```diff
@@ -72,6 +72,10 @@ WatchFaceAnalog::WatchFaceAnalog(Pinetime::Applications::DisplayApp* app,
   lv_label_set_text_static(plugIcon, Symbols::plug);
   lv_obj_align(plugIcon, nullptr, LV_ALIGN_IN_TOP_RIGHT, 0, 0);
 
+  bleIcon = lv_label_create(lv_scr_act(), nullptr);
+  lv_label_set_text_static(bleIcon, "");
+  lv_obj_align(bleIcon, nullptr, LV_ALIGN_IN_TOP_RIGHT, -30, 0);
```

Comment:
Aligning this to `OUT_LEFT_TOP` of the batteryIcon with offset 5 I think would be the same position, but a bit more correct.

Reply:
I did that (the offset to make it look the same was -18), but I accidentally committed it along with the merge of develop into my fork. Is that ok or should I somehow squash all the commits in this pr into a single one, and if so how do I do it?

Reply:
https://github.com/InfiniTimeOrg/InfiniTime/blob/develop/CONTRIBUTING.md#commit-conventions Clean merge commits are fine, they'll be automatically squashed, however conflicts should be resolved by rebasing. When on this branch you run `git rebase -i develop`, where develop is a branch that has the latest changes from develop in this repo. The exact branch name may depend on your local setup. I suggest reverting this change and we'll merge it as it was previously.

Reply:
I reverted the commit using the `OUT_LEFT_TOP`.

Reply:
The last commit is a mistake. You should rebase again and drop the commit.

Reply:
done.